src/components/Permissions/DataList.jsx

```diff
@@ -1,15 +1,114 @@
+import CozyClient, {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
 
-const DataList = () => {
-  const { t } = useI18n()
+const DataList = ({ t }) => {
+  const THIRTY_SECONDS = 30 * 1000
+  const queryResultApps = useQuery(Q(APPS_DOCTYPE), {
+    as: APPS_DOCTYPE,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
+  })
+  const queryResultKonnectors = useQuery(Q(KONNECTORS_DOCTYPE), {
+    as: KONNECTORS_DOCTYPE,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
+  })
+  const hiddenSlugs = ['home', 'store', 'settings']
+  let queryResultSlugs = []
+  let permissionsToDisplay = {}
+
+  if (
+    queryResultApps?.data?.length > 0 &&
+    queryResultKonnectors?.data?.length > 0
+  ) {
+    queryResultSlugs = [
+      ...queryResultApps?.data,
+      ...queryResultKonnectors?.data
+    ].filter(({ slug }) => !hiddenSlugs.includes(slug))
+
+    for (let { slug, permissions } of queryResultSlugs) {
+      Object.values(permissions).map(({ type }) => {
+        if (!permissionsToDisplay[type]) {
+          permissionsToDisplay[type] = []
+        }
+        permissionsToDisplay[type].push(slug)
+      })
+    }
+  }
+  const sortPermissionsByName = permissionsToDisplay => {
+    return Object.entries(permissionsToDisplay).sort((a, b) => {
+      return t('CozyClient.Permissions.' + a[0])
+        .toString()
+        .localeCompare(t('CozyClient.Permissions.' + b[0]))
+    })
+  }
+  console.log(permissionsToDisplay)
   return (
     <Page narrow>
       <PageTitle>{t('Permissions.data')}</PageTitle>
+      {(isQueryLoading(queryResultApps) ||
+        isQueryLoading(queryResultKonnectors)) &&
+      (!hasQueryBeenLoaded(queryResultApps) ||
+        !hasQueryBeenLoaded(queryResultKonnectors)) ? (
+        <Spinner size="large" className="u-flex u-flex-justify-center u-mt-1" />
+      ) : queryResultApps.fetchStatus === 'failed' ||
+        queryResultKonnectors.fetchStatus === 'failed' ? (
+        <Typography variant="body1" className="u-mb-1-half">
+          {t('Permissions.failedRequest')}
+        </Typography>
+      ) : (
+        <List>
+          {sortPermissionsByName(permissionsToDisplay).map(([key, slugs]) => {
+            const iconName = getPermissionIconName(key)
+            return (
+              <div key={key}>
+                <Link to="/">
+                  <ListItem button>
+                    <ListItemIcon>
+                      <Icon
+                        icon={
+                          require(`cozy-ui/transpiled/react/Icons/${iconName}`)
+                            .default
+                        }
+                        size={mediumSize}
+                      />
+                    </ListItemIcon>
+                    <ListItemText
+                      primary={t('CozyClient.Permissions.' + key)}
+                      secondary={t('Permissions.numberOfApplications', {
+                        smart_count: slugs.length
```

Comment:
we need to test:
- translation plural and singular
- icon is present or not
- apps + konnectors are present
- permissions are sorted by name
- a case with 1 permission is coming from 2 apps/konnectors

i don't see enough tests, i think